boards/arm/atsamr21_xpro/doc/index.rst

```diff
@@ -0,0 +1,208 @@
+.. _atsamr21_xpro:
+
+SAM R21 Xplained Pro Evaluation Kit
+###################################
+
+Overview
+********
+
+The SAM R21 Xplained Pro is a compact evaluation board by Atmel featuring a
+ATSAMR21G18A SoC. The SoC includes a SAMR21 ARM Cortex-M0+ micro-controller
+bundled with Atmel's AT86RF233, a 2.4GHz IEEE802.15.4 compatible radio.
+The kit includes Atmel’s Embedded Debugger (EDBG), which provides a full
+debug interface without the need for additional hardware.
+
+.. image:: img/atsamr21_xpro.png
+     :width: 500px
+     :align: center
+     :alt: ATSAMR21-XPRO
+
+Hardware
+********
+
+- ATSAMR21G18A ARM Cortex-M0+ processor at 48 MHz
+- 32.768 kHz crystal oscillator
+- 256 KiB flash memory and 32 KiB of RAM
+- One yellow user LED
+- One mechanical user push button
+- One reset button
+- On-board USB based EDBG unit with serial console
+
+Supported Features
+==================
+
+The atsamr21_xpro board configuration supports the following hardware
+features:
+
++-----------+------------+--------------------------------------+
+| Interface | Controller | Driver/Component                     |
++===========+============+======================================+
+| NVIC      | on-chip    | nested vector interrupt controller   |
++-----------+------------+--------------------------------------+
+| SYSTICK   | on-chip    | systick                              |
++-----------+------------+--------------------------------------+
+| WDT       | on-chip    | Watchdog                             |
++-----------+------------+--------------------------------------+
+| GPIO      | on-chip    | I/O ports                            |
++-----------+------------+--------------------------------------+
+| USART     | on-chip    | Serial ports                         |
++-----------+------------+--------------------------------------+
+| SPI       | on-chip    | Serial Peripheral Interface ports    |
++-----------+------------+--------------------------------------+
```

Comment:
Commit message states that "only UART, debug LED and user button are supported". Here we have also WDT, SPI.

Reply:
s/supported/tested/